packages/garbo/src/lib.ts

```diff
@@ -198,10 +198,8 @@ export const gooseDroneEligible = () =>
   have($familiar`Grey Goose`);
 
 export function averageTargetNet(): number {
-  const goose = gooseDroneEligible() ? 2 : 1;
-
   return targettingItems()
-    ? valueDrops(globalOptions.target) * goose
+    ? valueDrops(globalOptions.target)
```

Comment:
How is this related to the current changes?

Reply:
We can no longer assume we'll use gooso, so we cannot add that multiplier to determine the benefit of fighting target. The modifier existed in tandem with the cost in Fight.ts that guaranteed gooso usage.